src/v/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 add_compile_options(-Wall)
+add_compile_definitions(FMT_DEPRECATED_OSTREAM)
```

Comment:
I guess this could be added as exported by the target.

Reply:
you mean by fmt? we don't control that

Reply:
> you mean by fmt? we don't control that

I don't know if you can do it with `fetch_deps`. I guess the flags here are used to compile everything, so it should be fine.

Reply:
oh i see just add it where it's used? gotcha. yeh i'd be shocked if anything less than like 99% of our v_library() didn't have some file that used fmt/operator<< either directly or via seastar::log, but cases of which i guess need this definition to be set.